src/Microsoft.Azure.WebJobs.Host/Config/JobHostRetryOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// Configuration options for controlling function execution retry behavior.
+    /// </summary>
+    public class JobHostRetryOptions
+    {
+        /// <summary>
+        /// The retry count.
+        /// </summary>
+        public int RetryCount { get; set; }
+
+        /// <summary>
+        /// The duration to wait for for a particular retry attemp
+        /// </summary>
+        public TimeSpan SleepDuration { get; set; } = TimeSpan.Zero;
```

Comment:
I'd call this "Delay" - the delay between retries.

Reply:
Updated!